README.md

```diff
@@ -113,8 +129,6 @@ The first jobs will be submitted and should now be queued or running. Note, you
 to run rocotorun until the workflow is complete. This can be done via cron jobs or manually submitting 
 rocotorun until the workflow is complete.
 
-Rocotodocumentation can be found here: https://github.com/christopherwharrop/rocoto/wiki/documentation
```

Comment:
Is there a reason this is taken out?

Reply:
This information is already included twice in the README.